File: 3199-distribute-candies-among-children-i/distribute-candies-among-children-i.py
class Solution:
    def distributeCandies(self, n: int, limit: int) -> int:
        candyDistCount = 0
        for i in range(limit+1):
            for j in range(limit+1):
                for k in range(limit+1):
                    if i+j+k == n:candyDistCount+=1
        return candyDistCount
        

File: 3199-distribute-candies-among-children-i/distribute-candies-among-children-i.cpp
class Solution {
public:
    int distributeCandies(int n, int limit) {
        int candyDistCount = 0;
        int i,j,k;
        for(i=0; i<=limit; i++){
            for(j=0; j<=limit; j++){
                for(k=0; k<=limit; k++){
                    if(i+j+k == n){
                        candyDistCount+=1;
                    };
                };
            };
        };
        return candyDistCount;
    }
};